collections/_version2-english/9-fork/1-basic-knowledge.md

```diff
@@ -0,0 +1,119 @@
+{% assign glossary_market = glossary_path | append: '#Market' %}
+{% assign glossary_report = glossary_path | append: '#Report' %}
+{% assign glossary_initial_reporter = glossary_path | append: '#Initial_Reporter' %}
+{% assign glossary_initial_report = glossary_path | append: '#Initial_Report' %}
+{% assign glossary_dispute_round_phase = glossary_path | append: '#Dispute_Round_Phase' %}
+{% assign glossary_outcome = glossary_path | append: '#Outcome' %}
+{% assign glossary_tentative_outcome = glossary_path | append: '#Tentative_Outcome' %}
+{% assign glossary_dispute_round = glossary_path | append: '#Dispute_Round' %}
+{% assign glossary_dispute = glossary_path | append: '#Dispute' %}
+{% assign glossary_reputation_token = glossary_path | append: '#Reputation_Token' %}
+{% assign glossary_dispute_bond = glossary_path | append: '#Dispute_Bond' %}
+{% assign glossary_universe = glossary_path | append: '#Universe' %}
+{% assign glossary_final_outcome = glossary_path | append: '#Final_Outcome' %}
+{% assign glossary_forking_market = glossary_path | append: '#Forking_Market' %}
+{% assign glossary_creation_bond = glossary_path | append: '#Creation_Bond' %}
+{% assign glossary_parent_universe = glossary_path | append: '#Parent_Universe' %}
+{% assign glossary_child_universe = glossary_path | append: '#Child_Universe' %}
+{% assign glossary_all_theoretical_rep = glossary_path | append: '#All_Theoretical_REP' %}
+{% assign glossary_forking_period = glossary_path | append: '#Forking_Period' %}
+{% assign glossary_winning_universe = glossary_path | append: '#Winning_Universe' %}
+{% assign glossary_losing_universe = glossary_path | append: '#Losing_Universe' %}
+
+# What is a fork?
+[Forking]({{glossary_fork}}) is the last [market]({{glossary_market}}) resolution method. It is a very disruptive process and is intended to be a rare occurrence.
+
+After a market enters [reporting]({{glossary_report}}), its [initial reporter]({{glossary_initial_reporter}}) submits [initial report]({{glossary_initial_report}}), and the market enters the [dispute round phase]({{glossary_dispute_round_phase}}). and the reported [outcome]({{glossary_outcome}}) becomes the market's [tentative outcome]({{glossary_tentative_outcome}}).
+During a [dispute round]({{glossary_dispute_round}}), any [REP]({{glossary_reputation_token}}) holder may [dispute]({{glossary_dispute}}) the tentative outcome by staking REP on an alternative outcome of the market which they consider the "truth". To be successful the dispute, their staking REP needs to reach a certain amout. This amount is called [dispute bond]({{glossary_dispute_bond}}). With each successive dispute round, a higher dispute bond is needed. If the size of dispute bond reaches a certain [threshold](#how-the-market-enters-the-fork-state), the [universe]({{glossary_universe}}) where the diputed market exists is split into multiple universe, one for each possible outcome of the disputed market. This split is the fork.
```

Comment:
> into multiple universe,

should be
> into multiple universes,

Reply:
```suggestion
During a [dispute round]({{glossary_dispute_round}}), any [REP]({{glossary_reputation_token}}) holder may [dispute]({{glossary_dispute}}) the tentative outcome by staking REP on an alternative outcome of the market which they consider the "truth". To be successful the dispute, their staking REP needs to reach a certain amount. This amount is called [dispute bond]({{glossary_dispute_bond}}). With each successive dispute round, a higher dispute bond is needed. If the size of dispute bond reaches a certain [threshold](#how-the-market-enters-the-fork-state), the [universe]({{glossary_universe}}) where the disputed market exists is split into multiple universes, one for each possible outcome of the disputed market. This split is the fork.
```

Reply:
I'm closing this conversation. Since those were fixed by the pull request https://github.com/msagansk/Augur.Guide/pull/28.